### ISSUE TITLE
Games DO run on Windows, SAME EXACT games do NOT run on Apple M2 Silicone

### ISSUE THREAD

**Issue by Dochartaigh** [xemu-project/xemu#1392]
### Bug Description

I posted about a couple issues with specific games not running on my 2023 Mac mini M2 Pro / Apple Silicone ([1st](https://github.com/xemu-project/xemu/issues/1389), [2nd](https://github.com/xemu-project/xemu/issues/1386), [3rd](https://github.com/xemu-project/xemu/issues/1387)). 

The developer/mod? closed a couple of those issues and said that I should dump my "personal game copy and try again". I assumed that meant they think my game copy is bad, and that's the issue... so I dug deeper into this. Turns out the SAME EXACT file that does NOT run on my Mac M2 Pro, DOES run on my Windows PC – so this seems to be an issue specific to the Mac version of Xemu only.

I filmed two tests that I hope help show this issue. I'm TOTALLY open to help with anything to get this fixed! If you need logs, need me to test any other game versions (or convert rips of my physical discs to XISO in a different/new way), test different Xemu versions, whatever - just let me know! Totally psyched to be able to emulate some later systems on my new Mac (which Mac is always severely lacking with emulation for later gen consoles!), so glad to help any way I can.

Anyway, here's the two videos. I show the 2x problem game files MD5 hashes first, then load them up on both my Windows PC and my Mac:

[Here's the video of it running on Windows PC](https://www.youtube.com/watch?v=faEx7WUqGH0)
[Here's the video of it NOT running on Mac M2 Apple Silicone](https://www.youtube.com/watch?v=Xn7ISEoQ8zU)

### Expected Behavior

The same games to run both on my Windows PC, and on my Mac M2 / Apple Silicone

### xemu Version

Mac: v0.7.83
Windows: v0.7.83

### System Information

2023 Mac mini M2 Pro (10/16-core). Ventura 13.2

i7700 Intel, 8gb RAM, ancient GeForce GT 1030 2gb. Up-to-date Windows 10.

**Comment by domrozum** [comment]
Just going to throw out there that these issues probably apply to the Mac client in general... I have had some issues with some games like Jet Set Radio that affect gameplay on Mac, but run perfectly on Windows. I'd have to test out all of the games you've mentioned in the last week on my own Mac, which utilizes an Intel processor instead of Apple Silicon. It'd be great if anything can be done with the Mac clients to make these games run exactly like their Windows counterparts.

**Comment by Dochartaigh** [comment]
> Just going to throw out there that these issues probably apply to the Mac client in general... 

Yup, that's why I said "this seems to be an issue specific to the Mac version of Xemu only" - since it's fine on my PC. Didn't feel like pulling out my measly 1.1ghz Intel Mac to test on there, but PLEASE post if you give it a whirl and if it works or not.

**Comment by abaire** [comment]
Looks like two of the three original bugs were reopened, I'm going to dup this against the KOTOR bug and reopen it since this issue doesn't link to any specific titles and thus is easily lost.
